src/meta/play-meta.js

```diff
@@ -212,5 +213,18 @@ export const plays = [
     blog: '',
     video: '',
     language: 'ts'
+  }, {
+    id: 'pl-netlify-card-game', 
+    name: 'Netlify Card Game', 
```

Comment:
Why is it called Netlify Card Game? I am just curious. To me, it sounds like a cool Memory Game. 😀